dcicutils/ecs_utils.py

```diff
@@ -7,39 +7,39 @@ class ECSUtils:
     """ Utility class for interacting with ECS - mostly stubs at this point, but will likely
         expand a lot.
     """
-    WSGI = 'wsgi'  # XXX: might want to be joined with an identifier?
-    INDEXER = 'indexer'
-    INGESTER = 'ingester'
-    SERVICES = [
-        WSGI, INDEXER, INGESTER
-    ]
 
-    def __init__(self, *, cluster_name):
+    def __init__(self):
         """ Cluster name in this case is the env name """
-        self.cluster_name = cluster_name
         self.client = boto3.client('ecs', region_name=CGAP_ECS_REGION)  # same as ECR
 
-    def update_ecs_service(self, *, service_name=WSGI):
+    def list_ecs_clusters(self):
```

Comment:
This looks like an incompatible change. Needs a major version bump. See my comments on pyproject.toml.